app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1> CodeCraft Info statistics </h1>
+
+<h1> Total number of participants per edition</h1>
+<%= render 'charts/x_y_chart', editions: @participants_per_edition, chart_id: "chartdiv_xy", value: { name: "name", number: "number" } %>
```

Comment:
Please rename this partial using the name of the chart as it is used by AmChart library, `_simple_column` I suppose. Because we will place all these partials inside charts folder, the `_chart` can be removed, it's quite clear that we refer to a chart, in this case `_simple_column.html.erb` should be enough.
Another aspect to be considered here is the generalization aspect of those partials, so please try to avoid the use of `editions` or other names that give a more specialized way of naming. For this example, probably `data` fits better.